dutch/bos/verlaten-weg/groot-monster/groot-monster

```diff
@@ -0,0 +1,3 @@
+Ai, het was de verkeerde richting. Je stopt abrupt want je ziet 2 grote boze gele ogen naar je kijken.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.